UrologicalSupplies/R4/resources/Questionnaire-R4-UrologicalSupplies.json

```diff
@@ -0,0 +1,361 @@
+            }
+          ],
+          "item": [
+            {
+              "linkId": "5.3.1",
+              "type": "choice",
+              "text": "Order",
+              "answerValueSet": "http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1219.62"
+            },
+            {
+              "linkId": "5.3.2",
+              "text": "Item Description",
+              "type": "string"
+            },
+            {
+              "linkId": "5.3.3",
+              "text": "Usual Max/Mo.",
+              "type": "string"
+            },
+            {
+              "linkId": "5.3.4",
+              "text": "Quantity",
+              "type": "string"
+            },
+            {
+              "linkId": "5.3.5",
+              "text": "Refills",
+              "type": "string"
+            }
+          ],
+          "repeats": "true",
+          "answerOption": []
+        },
+        {
+          "required": false,
+          "linkId": "5.4",
+          "text": "Order: Miscellaneous urological supplies",
+          "type": "group",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-itemControl",
+              "valueCodeableConcept": {
+                "coding": [
+                  {
+                    "system": "http://hl7.org/fhir/questionnaire-item-control",
+                    "code": "gtable"
+                  }
+                ]
+              }
+            }
+          ],
+          "item": [
+            {
+              "linkId": "5.4.1",
+              "type": "choice",
+              "text": "Order",
+              "answerValueSet": "http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1219.63"
+            },
+            {
+              "linkId": "5.4.2",
+              "text": "Item Description",
+              "type": "string"
+            },
+            {
+              "linkId": "5.4.3",
+              "text": "Usual Max/Mo.",
+              "type": "string"
+            },
+            {
+              "linkId": "5.4.4",
+              "text": "Quantity",
+              "type": "string"
+            },
+            {
+              "linkId": "5.4.5",
+              "text": "Refills",
+              "type": "string"
+            }
+          ],
+          "repeats": true,
+          "answerOption": []
+        },
+        {
+          "required": false,
+          "linkId": "5.5",
+          "text": "Other Order",
+          "type": "group",
+          "repeats": true,
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-itemControl",
+              "valueCodeableConcept": {
+                "coding": [
+                  {
+                    "system": "http://hl7.org/fhir/questionnaire-item-control",
+                    "code": "gtable"
+                  }
+                ]
+              }
+            }
+          ],
+          "item": [
+            {
```

Comment:
We need a text box for the provider to add the "other" order before description
![Screen Shot 2020-08-10 at 3 10 14 PM](https://user-images.githubusercontent.com/58572522/89820986-d907fa00-db1b-11ea-893b-e1974a0bf4d6.png)